### PR TITLE
[BUG] 0.5.2 bugs

### PR DESCRIPTION
**Pull request by Rahul-Vijay**
- Fix qty BUG

### CODE REVIEW

**Review by vikasprashar99 — APPROVED** [review]
**addition and remove bugs are fixed** but on clearing the  app from ram the cart items are not visible but when i add a new item only then they are there

**Comment by Rahul-Vijay** [comment]
- Now you can close the app and open again, cart items will be present

File: lib/classes/cartItems_box.dart [hunk 1]
```diff
@@ -58,6 +58,7 @@ class CIBox {
       aquantity.id = aq.id;
       aquantity.value = aq.value;
       aquantity.metric = aq.metric;
+      if (aq.qty > 1) aq.qty = 1;
```
Comment (Rajat012k) [inline lib/classes/cartItems_box.dart:61]: @Rahul-Vijay is this for handling the qty when it is 1?

**Review by Rajat012k — APPROVED** [review]
LGTM!

File: lib/classes/cartItems_box.dart (hunk 1 above)
Comment (Rahul-Vijay) [reply lib/classes/cartItems_box.dart:61]: - This function runs when 1st time an object is added which doesn't exist in cart.

When you add and then remove. Then add again, qty would be 2, or 3 or 4 depending on number of times you pressed on minus. So if it is greater than 1, means it was previously added then removed. So we reset it back to 1.

**Pull request merged**